src/css/styles.css

```diff
@@ -1,25 +1,27 @@
 @import './diff.css';
 
+/* ===== Global ===== */
 body {
     background: #136a8a;
     background: linear-gradient(45deg, #136a8a, #267871);
 }
 
+.fa {
+    margin-right: 0.5em;
+}
+
+/* ===== Containers ===== */
 .container-list-view, .container-page-view {
     background: white;
-    border-radius: 3px;
     color: #444;
-    margin: 0 20px;
+    margin: 5px 0 ;
 }
 
 .container-page-view {
-    padding: 20px;
-}
-
-.lnk-action {
-    margin-right: 17px;
+    padding: 1em;
```

Comment:
I changed the padding to be 15px, so we wouldn't have to add another rule to fight Bootstrap. Less to get rid of when we migrate away as well.